CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+#👉 How to contribute to Giskard?
+
+Everyone is encouraged to contribute, and we appreciate each and every one of them. Helping the community is thus not limited to writing code. The community will greatly benefit from your efforts to provide clarification, assist others, and enhance the documentation. 📗
+
+Additionally, it is helpful if you help us spread the word by mentioning the library in blog articles about the amazing projects it enabled, tweeting about it on occasion, or just staring the repository! ⭐️
+
+If you choose to contribute, please be mindful to respect our [code of conduct](https://github.com/Giskard-AI/giskard/blob/main/CODE_OF_CONDUCT.md).
+
+## The different ways you can contribute to Giskard!
+
+There are 4 ways you can contribute to Giskard:
+* Fixing outstanding issues with the existing code;
+* Implementing new tests;
+* Contributing to the examples or to the documentation;
+* Submitting issues related to bugs or desired new features.
+
+
+### Did you find a bug? 
+
+First, we would really appreciate it if you could **make sure the bug was not
```

Comment:
That would suppose our bug tracker is public on Github. I really don't think the current tool youtrack is fit for an open-source community.

Reply:
ok let's discuss it together about whether YouTrack is adapted to open source or not